README.md

```diff
@@ -321,12 +321,14 @@ specific seed data files you need for integration testing.
 This macro is intended to be used as a `run-operation` when generating Fivetran dbt source package staging models/macros. This macro will receive user input to create all necessary ([bash commands](columns_setup.sh)) appended with `&&` so they may all be ran at once. The output of this macro within the CLI will then be copied and pasted as a command to generate the staging models/macros.
 **Usage:**
 ```bash
-dbt run-operation staging_models_automation --args '{package: asana, source_schema: asana_source, source_database: database-source-name, tables: ["user","tag"]}'
+dbt run-operation staging_models_automation --args '{package: apple_search_ads, source_schema: apple_search_ads, source_database: dbt-package-testing, tables: ["campaign_history","campaign_report"]}'
```

Comment:
```suggestion
dbt run-operation staging_models_automation --args '{package: apple_search_ads, source_schema: apple_search_ads, source_database: database-source-name, tables: ["campaign_history","campaign_report"]}'
```

Let's keep this as `database-source-name` just in case someone ends up wanting to use this for another warehouse. Then they will know that they need to change this to be the database or project name.